README.md

```diff
@@ -1,2 +1,30 @@
 # liveDVR
 DVR implementation built on top of live streams
+
+# Installation #
+
+## Pre-requisites ##
+
+* Make sure node is installed
+* Install npm
+* Install nvm (curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash)
+* Clone repository
+* C++ compiler
+* Wowza + media-server installed and you are able to stream to it.
+
+## Steps to install ##
+
+* Go to repository and run `nvm install;nvm use`
+* Run command `npm install`
+* cd into "build_scripts"
+* Run `./build_ffmpeg.sh <location you want ffmpeg to be> <ffmpeg version>`
+    - NOTE: While this script supports any ffmpeg version we reccomend using "3.0". (no need for 'n' in beginning of version name)
+* Run `./build_ts2mp4_convertor.sh <ffmpeg location> <repository path>/liveRecorder/`
```

Comment:
@lilachmaliniak - Noam's usage here is far more coherent then what the scripts currently have, change the actual usage message accordingly.
Also, whether than "repository path", better use "livedvr clone/source path", which is clearer IMHO.